docs/GettingStarted.md

```diff
@@ -880,6 +882,44 @@ alternate_client = Demo::Echo::Service.rpc_stub_class.new(
 
 The integration will ensure that the `configured_interceptor` establishes a unique tracing setup for that client instance.
 
+### HTTParty
+
+The `httparty` integration adds instrumentation for sending HTTP requests using HTTParty.
+
+To setup your integration, use the `Datadog.configure` method like so:
+
+```ruby
+require 'httparty'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.use :httparty, options
+end
+```
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | ----------- | ------- |
+| `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
+| `distributed_tracing` | Enables [distributed tracing](#distributed-tracing) | `true` |
+| `service_name` | Service name for `httparty` instrumentation. | `'httparty'` |
+| `tracer` | `Datadog::Tracer` used to perform instrumentation. Usually you don't need to set this. | `Datadog.tracer` |
+
+In situations where you have multiple clients calling multiple distinct services, you may configure the clients like so:
+
+```ruby
+class FooClient
+  include HTTParty
+  base_uri 'https://example.com'
+  dd_options service_name: 'foo_service'
```

Comment:
This is really cool.
I'm not 100% sure on the naming here (`dd_options`).
I think a third opinion would be helpful regarding such naming.

Reply:
I think `ddtrace_options` would help us avoid possible conflicts in the future, specially with Datadog's own `datadog/statsd` gem.

One other option is to have it as only `ddtrace`, like `ddtrace service_name: 'foo_service'`, but I think this approach doesn't communicate the intention of the method enough.

Having a configuration block, instead of a hash is also in line with our other configuration patterns `ddtrace_options do |o| o.service_name = 'foo_service' end`, but it seems like it would make the user's code less readable.

Reply:
I'll update the name to `ddtrace_options`.